pilot/docker/envoy_ingress.yaml

```diff
@@ -0,0 +1,154 @@
+# Ingress configuration using Envoy v2 bootstrap.
+#
+# Has 'zero vpn' port forwarding for the control plane - allowing machines outside the VPN with proper certificates
+# to connect to Istio.
+#
+# Current config has a sample port open, exposing an internal service. This will be rolled into LDS and automatically
+# exposed using service annotations in a future version.
+#
+# To customize the envoy config, create a config map and update the ingress config to use the custom envoy bootstrap
+# from the config map.
+#
+# Setting up SNI in LDSv2:
+# https://www.envoyproxy.io/docs/envoy/latest/faq/sni.html#faq-how-to-setup-sni
+stats_config:
```

Comment:
This is going to be very challenging to maintain given the amount of YAML present in this PR, and not quite stable API surface that it builds upon.

Please use templating or code to share as much as possible.

Reply:
Yes, this will be replaced by the template - the code to use template is in progress, until it is 
ready it's useful to test ( so I know what to put in the template )